parsers/html.c

```diff
@@ -20,6 +20,8 @@
 #include "keyword.h"
 #include "promise.h"
 
+#define MAX_DEPTH 1000
```

Comment:
AFTER merging this change, can I change this to HTML_MAX_DEPTH ?

Ctags has `--maxdepth` option. To avoid confusion, I would like to add a prefix.

```
[yamato@master]~% u-ctags --help | grep max
  --maxdepth=N
       Specify maximum recursion depth.
```